formula_renames.json

```diff
@@ -89,6 +89,7 @@
   "polarssl": "mbedtls",
   "postgresql94": "postgresql@9.4",
   "postgresql95": "postgresql@9.5",
+  "premake": "premake4",
```

Comment:
Is this enough to signal the change to the end-users?

Reply:
I believe so, but it should point at the newest version.

Reply:
It can't because the old formula was for premake4 in fact. You could install premake5 only from the source because it was set as formula head.